docs/developer-documentation.md

```diff
@@ -120,8 +120,40 @@ Provides the following factory methods:
   - `size`: file size in bytes.
   - `title` (optional): file title, defaults to file `name` if empty.
   - `language_code` (optional): two letter language code of the current file's language.
-- `FileValueObject::fromFileEntity(FileInterface $file_entity)`: accept an object implementing the
+
+- `FileValueObject::fromFileEntity(FileInterface $file_entity)`: accepts an object implementing the
   `\Drupal\file\FileInterface` interface.
 
+#### `GalleryItemValueObject`
+
+Used in the following patterns:
+
+- [`gallery`](../templates/patterns/gallery/gallery.ui_patterns.yml)
+
+Provides the following factory methods:
+
+- `DateValueObject::fromArray(array $values = [])`: accepts an array with the following properties:
+  - `thumbnail`: Thumbnail to be rendered on the gallery item.
```

Comment:
We should add here that this is a `ImageValueObject` so that the array should be compatible with `ImageValueObject::fromArray(array $values = [])`, so something on the line of:

>   - `thumbnail`: Thumbnail to be rendered on the gallery item. Check `ImageValueObject::fromArray(array $values = [])` for a list of acceptable array values.

Reply:
:+1: